integration-tests/actions/ocr2_helpers_local.go

```diff
@@ -272,3 +274,38 @@ func GetOracleIdentitiesWithKeyIndexLocal(
 
 	return S, oracleIdentities, eg.Wait()
 }
+
+func DeleteAllOCR2JobsAndBridges(nodes []*client.ChainlinkClient) error {
+	for _, node := range nodes {
+		if node == nil {
+			return fmt.Errorf("found a nil chainlink node in the list of chainlink nodes while tearing down: %v", nodes)
+		}
+		jobs, _, err := node.ReadJobs()
+		if err != nil {
+			return errors.Wrap(err, "error reading jobs from chainlink node")
```

Comment:
Let's use std lib errors here:
```suggestion
			return fmt.Errorf("error reading jobs from chainlink node: %w", err)
```
Trying to move away from pkg/errors since it's not necessary or even maintained any more.

Reply:
Oh I though the import was new, but I guess there is precedent here for local style. Either way in that case :shrug: